### ISSUE TITLE
Warn about conflicts between update sites

### ISSUE THREAD

**Issue by hinerm** [imagej/imagej-ui-swing#36]
When update sites ship modified versions of the same jar, it would be nice if we could issue some sort of warning about potential incompatibility/conflicts.

**Comment by ctrueden** [comment]
It's tricky without special-casing certain update sites. In particular, many update sites override some files from the `Fiji` site, and this is just fine. The issue is when there are two "third party" update sites that both override the same file—then it becomes less clear which one should "win", or really if either one taking precedence will result in a mutually compatible state.

**Comment by hinerm** [comment]
@ctrueden agreed. Sorry, I meant "non-core update sites" in the original description. And because of that ambiguity I propose just issuing warnings about the potential for problems, instead of trying to actually determine if state is compatible.
